### ISSUE TITLE
addurls errors out on some urls, leading to "no commit"

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#2333]
#### What is the problem?
```
$> datalad --report-status failure run datalad --report-status failure addurls --nosave .datalad/crawl/phenotype_mri.csv '{MRIs}' '{project}/sub-{participant_id}/ses-{session_id}/{_url_filename_root}_rep-{_repindex}{_url_filename_ext}'
[INFO   ] == Command start (output follows) =====
[WARNING] Dropped 2926 row(s) that had an empty URL
addurls(error): /mnt/btrfs/datasets/datalad/crawl/labs/openneurolab/metasearch/corr/sub-27223/ses-1/T1_rep-0.mgz (file) [AnnexBatchCommandError: command 'addurl'
Error, annex reported failure for addurl (url='https://s3.amazonaws.com/fcp-indi/data/Projects/CORR/Outputs/IBA_TRT/freesurfer/0027223-session_1/mri/T1.mgz'): {'command': 'addurl', 'file': None, 'success': False} [annexrepo.py:add_url_to_file:2086]]
addurls(error): /mnt/btrfs/datasets/datalad/crawl/labs/openneurolab/metasearch/corr/sub-27223/ses-1/T1_rep-1.mgz (file) [AnnexBatchCommandError: command 'addurl'
Error, annex reported failure for addurl (url='https://s3.amazonaws.com/fcp-indi/data/Projects/CORR/Outputs/IBA_TRT/freesurfer_gpu/0027223-session_1/mri/T1.mgz'): {'command': 'addurl', 'file': None, 'success': False} [annexrepo.py:add_url_to_file:2086]]
addurls(error): /mnt/btrfs/datasets/datalad/crawl/labs/openneurolab/metasearch/corr/sub-27224/ses-1/T1_rep-0.mgz (file) [AnnexBatchCommandError: command 'addurl'
Error, annex reported failure for addurl (url='https://s3.amazonaws.com/fcp-indi/data/Projects/CORR/Outputs/IBA_TRT/freesurfer/0027224-session_1/mri/T1.mgz'): {'command': 'addurl', 'file': None, 'success': False} [annexrepo.py:add_url_to_file:2086]]
addurls(error): /mnt/btrfs/datasets/datalad/crawl/labs/openneurolab/metasearch/corr/sub-27224/ses-1/T1_rep-1.mgz (file) [AnnexBatchCommandError: command 'addurl'
Error, annex reported failure for addurl (url='https://s3.amazonaws.com/fcp-indi/data/Projects/CORR/Outputs/IBA_TRT/freesurfer_gpu/0027224-session_1/mri/T1.mgz'): {'command': 'addurl', 'file': None, 'success': False} [annexrepo.py:add_url_to_file:2086]]

```

#### What version of DataLad are you using (run `datalad --version`)? On what operating system (consider running `datalad wtf`)?
master branch  0.9.3-840-gffec2642

**Comment by kyleam** [comment]
Hmm seem to be 404s.

**Comment by kyleam** [comment]
The "no commit" part should now be resolved by #2362.
